src/reference/asciidoc/logging.adoc

```diff
@@ -168,12 +168,29 @@ The following example shows how to configure a Log4j 2 appender:
         applicationId="myAppId" routingKeyPattern="%X{applicationId}.%c.%p"
         contentType="text/plain" contentEncoding="UTF-8" generateId="true" deliveryMode="NON_PERSISTENT"
         charset="UTF-8"
-        senderPoolSize="3" maxSenderRetries="5">
+        senderPoolSize="3" maxSenderRetries="5"
+        addMdcAsHeaders="true">
     </RabbitMQ>
 </Appenders>
 ----
 ====
 
+.Log4j 2 Appender Properties
+[cols="2l,2l,4", options="header"]
+|===
+| Property
+| Default
+| Description
+
+| `addMdcAsHeaders`
+| `false`
```

Comment:
We don't need these backticks since the columns are marked as `literal` - I pushed a commit this morning to fix this; so you might need to rebase.

You mentioned you are also working on the logback appender. Should we wait for that before merging?

Thanks for your work!

Reply:
Hey, I think yes - I've already updated it and implementing and troubleshooting tests

Reply:
now it's done, I think